website/users/forms.py

```diff
@@ -17,4 +17,23 @@ def clean_username(self):
         :return: the cleaned username variable
         """
         username = self.cleaned_data.get("username")
-        return quote(username)
+        return quote(username.lower())
+
+
+class AccountForm(forms.Form):
+    """Form for changing account details."""
+
+    username = forms.CharField(
+        label="Username",
+        help_text="This is your username used to log you in to this website.",
+        widget=forms.TextInput(attrs={"readonly": "readonly"}),
+    )
+    first_name = forms.CharField(
+        label="First name", widget=forms.TextInput(attrs={"readonly": "readonly"}),
+    )
+    last_name = forms.CharField(
+        label="Last name", widget=forms.TextInput(attrs={"readonly": "readonly"}),
+    )
+    email = forms.CharField(
```

Comment:
I think there is an email field for this? At least we should add an email validator

Reply:
The user details can not be changed, it is only for displaying.